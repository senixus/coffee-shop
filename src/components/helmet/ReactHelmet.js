import { Helmet } from "react-helmet";

const ReactHelmet = ({ title }) => {
  return (
    <>
      <Helmet>
        <meta charSet="utf-8" />
        <title>{title} | Coffee Shop</title>
      </Helmet>
    </>
  );
};

export default ReactHelmet;
