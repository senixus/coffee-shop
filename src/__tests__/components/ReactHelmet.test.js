import { shallow } from "enzyme";
import React from "react";
import ReactHelmet from "../../components/helmet/ReactHelmet";

it("expect to render ReactHelmet component", () => {
  const mockTitle = "Coffee";
  const wrapper = shallow(<ReactHelmet title={mockTitle} />);

  expect(wrapper.length).toBe(1);

  expect(wrapper).toMatchSnapshot();
});
