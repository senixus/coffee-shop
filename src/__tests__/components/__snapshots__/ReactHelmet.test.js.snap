// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render ReactHelmet component 1`] = `
<HelmetWrapper
  defer={true}
  encodeSpecialCharacters={true}
>
  <meta
    charSet="utf-8"
  />
  <title>
    Coffee
     | Coffee Shop
  </title>
</HelmetWrapper>
`;
